target/linux/bcm27xx/bcm2712/config-6.6

```diff
@@ -29,6 +29,7 @@ CONFIG_ARM64_EPAN=y
 CONFIG_ARM64_ERRATUM_1165522=y
 CONFIG_ARM64_ERRATUM_1286807=y
 CONFIG_ARM64_ERRATUM_1463225=y
+CONFIG_ARM64_ERRATUM_3194386=y
```

Comment:
This is not needed for raspberry pi, this workaround is only needed for Cortex-X4.

Reply:
sorry, you were right the Cortex-A76 in the Raspberry Pi 5 is also affected.

Reply:
on the armsr target we should also activate this option.

Reply:
Done, see latest commit